REFERENCE.md

```diff
@@ -35,7 +35,7 @@
 
 rabbitmq
```

Comment:
This comes from the first line. I think it can be removed

Reply:
removed